src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/furtherevidence/actionfurtherevidence/ActionFurtherEvidenceAboutToStartHandler.java

```diff
@@ -47,14 +49,14 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
     }
 
     private void setFurtherEvidenceActionDropdown(SscsCaseData sscsCaseData) {
-        List<DynamicListItem> listOptions = getFurtherActionEvidenceItems(List.of(values()));
-
+        List<DynamicListItem> listOptions = getFurtherActionEvidenceItems(sscsCaseData);
         sscsCaseData.setFurtherEvidenceAction(new DynamicList(listOptions.get(0), listOptions));
     }
 
-    private List<DynamicListItem> getFurtherActionEvidenceItems(List<FurtherEvidenceActionDynamicListItems> items) {
-        return items.stream()
-            .filter(item -> !item.equals(ADMIN_ACTION_CORRECTION) || isPostHearingsEnabled)
+    private List<DynamicListItem> getFurtherActionEvidenceItems(SscsCaseData sscsCaseData) {
+        return Stream.of(values())
+            .filter(item -> !item.equals(ADMIN_ACTION_CORRECTION)
+                || (!GAPS.equals(sscsCaseData.getSchedulingAndListingFields().getHearingRoute()) && isPostHearingsEnabled))
```

Comment:
This is missing test coverage
![image](https://github.com/hmcts/sscs-tribunals-case-api/assets/110455092/1138393f-9ca1-4a03-8a65-153849a85e52)

Reply:
added unit tests for CC